tests/console/system_prepare.pm

```diff
@@ -102,6 +103,7 @@ sub run {
 
     # stop and disable PackageKit
     quit_packagekit;
+    turn_off_gnome_screensaver_for_running_gdm;
```

Comment:
This test module is meant for console tests and should have nothing to do with X11 tests. AFAIR there is a better setup _x11 module

Reply:
In the summary of system_prepare.pm says: "Execute SUT changes which should be permanent", so it is not limited to console tests.

Reply:
@skliu We can create a new module such as 'setup_x11' with this method inside, and schedule it with condition-yaml of DESKTOP=gnome.

Reply:
Ok, I will try it.